emloop/tests/hooks/write_csv_test.py

```diff
@@ -37,7 +37,6 @@ def _get_epoch_data():
     return epoch_data
 
 
-@pytest.fixture
 def get_tmp_filename():
```

Comment:
@gdynusa why don't you just use the fixture? I feel like it would be better to employ `tmpdir` fixture a create a new fixture (e.g. `tmpfile`) using it. Btw there is also fixute `tmpdir_factory` which might be helpful (idk tbh).